examples/resources/opslevel_team_tag/resource.tf

```diff
@@ -0,0 +1,17 @@
+data "opslevel_team" "foo" {
```

Comment:
@rocktavious I added a new example file for the team_tag resource, but noticed all the other folder have a `import.sh` file. Is this also something that will be auto generated?

Reply:
nope that example need to be created too.  Its shows how to an existing team tag.